src/components/dialogs/move-dialog.js

```diff
@@ -28,7 +34,12 @@ const MoveDialog = ({ open, onClose, items }) => {
                     nbElements: items.length,
                 }
             )}
-            contentText={intl.formatMessage({ id: 'moveItemContentText' })}
```

Comment:
Why removing this property contentText ?

Reply:
oops I though you removed this prop. I will restore everywhere.